wordpress-mobile/WordPressAuthenticator-iOS
# UL&S: SiteCredentialsViewController: OnePassword function on username for iOS 12 users

Closes #322 

This PR adds the OnePassword functionality to the TextFieldTableViewCell for the site URL textfield. While testing on iOS 12 I discovered a bug where the SiteAddressViewController's Continue button would not enable after typing in a URL. I accidentally used a textfield delegate method that was available for iOS 13 only. So instead I created an IBAction in the TextFieldTableCell and attached it directly to the textfield. 

I also fixed a bug where the OnePassword icon was displaying in the wrong color on the old UI.

WPiOS Test PR: https://github.com/wordpress-mobile/WordPress-iOS/pull/14511

**Concerns**
@Gio2018 I'd like your opinion on this PR too, if you have a moment. I had to go back to using tuples to pass some of the actions from the textfield in TextFieldTableViewCell because a delegate method was not available in iOS 12. I'm not sure that splitting the textfield actions between the Cell and the ViewController is a good idea. I think it will be confusing to the next person who edits it, but I don't know what to do to make it clearer.

### To test
1. `bundle exec pod install` to ensure pods are up to date. If you get an error, try `bundle exec pod update WordPressKit`
2. Build and run (no errors)
3. Use the test PR https://github.com/wordpress-mobile/WordPress-iOS/pull/14511
